modules/odr_api/odr_api/api/endpoints/hugging_face.py

```diff
@@ -0,0 +1,45 @@
+import os
+from fastapi import APIRouter, File, UploadFile, HTTPException
+from huggingface_hub import HfApi
+from pathlib import Path
+import tempfile
+from io import BytesIO
+from odr_core.config import settings
+
+router = APIRouter(tags=["hugging_face"])
+
+
+@router.post("/hugging-face/upload-image")
+async def upload_to_huggingface(file: UploadFile = File(...)):
+    HF_TOKEN = settings.HF_TOKEN
+
+    if not HF_TOKEN:
```

Comment:
might be beneficial to add a note somewhere indicating the token needs write permissions